== SUBMITION (2019-05-14 20:01:43) 
-- CODE:
k = int(input("Leia o numero natural em k: "))
x = 0
y = 0

while ( y < k):
	x = x + 1 / factorial(y)
	y = y + 1
print(round(x, 8))
-- EXECUTION TIME:
0.147983
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1712/2499/main.py", line 6, in <module>
    x = x + 1 / factorial(y)
NameError: name 'factorial' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-14 20:02:38) 
-- CODE:
from math import*
k = int(input("Leia o numero natural em k: "))
x = 0
y = 0

while ( y < k):
	x = x + 1 / factorial(y)
	y = y + 1
print(round(x, 8))
-- OUTPUT:
2.71805556
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 20:02:41) 
-- CODE:
from math import*
k = int(input("Leia o numero natural em k: "))
x = 0
y = 0

while ( y < k):
	x = x + 1 / factorial(y)
	y = y + 1
print(round(x, 8))
-- EXECUTION TIME:
0.122187
-- TEST CASE 1:
---- input:
7
---- correct output:
2.71805556
---- user output:
2.71805556
-- TEST CASE 2:
---- input:
15
---- correct output:
2.71828183
---- user output:
2.71828183
-- TEST CASE 3:
---- input:
6
---- correct output:
2.71666667
---- user output:
2.71666667
-- GRADE:
100%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 20:03:30) 
-- CODE:
from math import*
k = int(input("Leia o numero natural em k: "))
x = 0
y = 0

while ( y < k):
	x = x + 1 / factorial(y)
	y = y + 1
print(round(x, 8))
-- EXECUTION TIME:
0.149803
-- TEST CASE 1:
---- input:
7
---- correct output:
2.71805556
---- user output:
2.71805556
-- TEST CASE 2:
---- input:
15
---- correct output:
2.71828183
---- user output:
2.71828183
-- TEST CASE 3:
---- input:
6
---- correct output:
2.71666667
---- user output:
2.71666667
-- GRADE:
100%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
